src/commands/subscription.ts

```diff
@@ -541,7 +597,7 @@ export async function tellMeWhenHandler(
 // -----------------------------------------------------------------------------
 export async function helpHandler(bot: SlackBot, message: CommandMessage) {
```

Comment:
Should we also have an explicit return type here that will help us catch if we don't return properly in the future?  Not necessary for this PR, but in general, I'd like to see us rely less on the type-inference in cases like this.

Reply:
I'm all in favor of more explicit typing

Reply:
Great. Same here.

Reply:
I don't mind type inference in callbacks, where it's obvious, but in top-level functions and the like, I prefer explicit typing.

Reply:
I think I'll open a ticket to go through all these handlers later, we should figure out what we want to do with them. Right now these callbacks are (bot, message) => void so we should probably decide if that's the paradigm we want to have in mind with them or if we want to clean them all up and start bubbling up promises for error handling and such